### PR TITLE
Add hostname MOTD banner

### PR DESCRIPTION
**Pull request by pfeerick**
Adding the hostname MOTD banner from the rock64

### CODE REVIEW

**Comment by ayufan** [comment]
Please don't depend on `figlet`. It will fail when installing `.deb` package manually. Instead, add `figlet` to base system dependencies (in make_rootfs.sh) and add a condition which checks if figlet is installed and shows prompty only when is.

**Comment by pfeerick** [comment]
Hm... we'll have to remove that on the rock64 then, as that is how you did it over there ;) It's already there in the base system dependencies, that was the only thing that had been missed when I added it (back when I added htop).

**Comment by ayufan** [comment]
Rock64 uses different update mechanism, using apt-get so adding
dependencies is safe then.

File: package/root/etc/update-motd.d/05-figlet [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ $(which figlet) ]; then
```
Comment (ayufan) [inline package/root/etc/update-motd.d/05-figlet]: `if which figlet 1>/dev/null 2>/dev/null; then`

We don't want to show figlet in motd.

Comment (pfeerick) [inline package/root/etc/update-motd.d/05-figlet]: We don't? :-P

Comment (ayufan) [inline package/root/etc/update-motd.d/05-figlet]: execution of `which figlet` which generates: :)

```
$ which figlet
figlet not found
$ which bash
/usr/local/bin/bash
```

Comment (pfeerick) [inline package/root/etc/update-motd.d/05-figlet]: I did commit the change incase you didn't notice ;)

Comment (ayufan) [inline package/root/etc/update-motd.d/05-figlet]: Still, I will be nitpicky, why `[ $(`? :)

Comment (pfeerick) [inline package/root/etc/update-motd.d/05-figlet]: Oh raspberry! It's your repo... be picky all you want :-P

**Pull request merged**